airflow_blogpost.md

```diff
@@ -0,0 +1,137 @@
+# Introduction
+
+Today, we're announcing an integration between Versatile Data Kit and Apache Airflow, a popular workload scheduler. This blog post serves to provide an overview of the relevant parts of these two projects, how they integrate, and how to set up an instance of Airflow which will trigger deployed VDK jobs.
+
+# Basic Concepts Overview
+
+Apache Airflow is "a platform to programmatically author, schedule and monitor workflows". It allows users to schedule workloads to be ran periodically, and also to build more complex workload structures by expressing them as directed acyclic graphs. The nodes of these graphs represent individual workloads, while the edges represent dependencies between workloads, meaning that a workload which is `pointed at` by another workload will only trigger if the previous workload completes successfully.
+Workloads within Airflow code are represented by operators. Operators are Python objects, which means that they are highly flexible in terms of what work is done by the actual workload. Airflow offers many different operators by default, and users can develop their own if they need more specialized behaviour, as we have done here.
+For more information on Apache Airflow, reference the official documentation here - https://airflow.apache.org/docs/apache-airflow/stable/index.html.
```

Comment:
```suggestion
For more information on Apache Airflow, reference [the official documentation here](https://airflow.apache.org/docs/apache-airflow/stable/index.html)
```

Use link syntax like above - no need to post the entire URL in the text Just makes it longer. This is just an example. I'll leave to you to fix the rest